# DP: Build libphobos on hppa-linux-gnu.

libphobos/

2020-04-27  Iain Buclaw  <ibuclaw@gdcproject.org>

	* configure.tgt: Add hppa-*-linux* as a supported target.

 
--- a/libphobos/configure.tgt
+++ b/libphobos/configure.tgt
@@ -30,6 +30,9 @@ case "${target}" in
   arm*-*-linux*)
 	LIBPHOBOS_SUPPORTED=yes
 	;;
+  hppa-*-linux*)
+	LIBPHOBOS_SUPPORTED=yes
+	;;
   mips*-*-linux*)
 	LIBPHOBOS_SUPPORTED=yes
 	;;
